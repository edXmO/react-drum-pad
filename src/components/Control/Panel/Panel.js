import React from 'react';
import '../../../css/App.css';
import Button from './Button/Button';

const Panel = () => {
    return (
        <div className='btn-panel'>
            <Button text={'ON / OFF'} type={'ON'} />
            <Button text={'+VOL'} type={'volume'} />
            <Button text={'-VOL'} tpye={'volume'} />
        </div>
    )
}

export default Panel;