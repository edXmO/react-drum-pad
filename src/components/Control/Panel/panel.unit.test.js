import React from 'react';
import renderer from 'react-test-renderer';


import Panel from '../Panel/Panel';

describe('should render all buttons without crashing', () => {
    let component = renderer.create(
        <Panel />
    );
    it('should render without crashing', () => {
        expect(component.toJSON()).toMatchSnapshot();
    });
});