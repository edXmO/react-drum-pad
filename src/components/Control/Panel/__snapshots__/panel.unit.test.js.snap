// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render all buttons without crashing should render without crashing 1`] = `
<div
  className="btn-panel"
>
  <div
    className="btn-panel__box "
  >
    <button
      className="btn-panel__btn"
      onClick={[Function]}
    />
    OFF
  </div>
  <div
    className="btn-panel__box "
  >
    <button
      className="btn-panel__btn"
      onClick={[Function]}
    />
    +VOL
  </div>
  <div
    className="btn-panel__box "
  >
    <button
      className="btn-panel__btn"
      onClick={[Function]}
    />
    -VOL
  </div>
</div>
`;
